CMakeLists.txt

```diff
@@ -156,11 +150,6 @@ include_directories(${SQLITE3_INCLUDE_DIRS})
 include_directories(${LIBDBUS_INCLUDE_DIRS})
 include_directories(${sodium_INCLUDE_DIR})
 
-if(BUILD_GENIVI)
-    add_subdirectory("third_party/rvi_lib")
-    set_property(TARGET rvi PROPERTY CXX_STANDARD 11)
-endif(BUILD_GENIVI)
-
 # Setup warnings. Do this after rvi_lib is added so that it isn't affected.
```

Comment:
Remove or modify this comment

Reply:
I did not know that secondaries uses it.  I added this folder for rvi support only.

Reply:
Ah yes, `git log` confirms that. Then we should just move the `secondary` directory outside of `ex` and get rid of `ex`.